subprojects/docs/src/docs/userguide/core-plugins/pmd_plugin.adoc

```diff
@@ -80,3 +80,20 @@ include::sample[dir="snippets/codeQuality/codeQuality/kotlin",files="build.gradl
 ====
 
 See the link:{groovyDslPath}/org.gradle.api.plugins.quality.PmdExtension.html[PmdExtension] class in the API documentation.
+
+[[sec:pmd_conf_threads]]
+=== Parallel analysis
+
+You can configure the number of threads to be used by PMD for running its analysis.
+
+====
+include::sample[dir="snippets/codeQuality/codeQuality/groovy",files="build.gradle[tags=pmd-threads]"]
+include::sample[dir="snippets/codeQuality/codeQuality/kotlin",files="build.gradle.kts[tags=pmd-threads]"]
+====
+
+[NOTE]
+====
+This configuration is internal to PMD and is not linked to <<performance#parallel_execution,the number of workers>> used by Gradle.
+It means that you have to pay attention to the value entered here and make sure it still makes sense in a multi project.
+Because parallel Gradle execution could result in different PMD tasks from different project running in parallel, having a multiplier effect on the configured value.
```

Comment:
```suggestion
This is because parallel Gradle task execution could result in different PMD tasks from different projects running in parallel.  If multiple PMD tasks execute simultaneously in n projects, then up to a maximum of (n * `thread`) PMD threads could run at the same time.
```